src/notification.test.ts

```diff
@@ -455,9 +532,14 @@ describe("BaseNotification", () => {
           features: {},
         });
 
-      await expect(
-        notification.fetchNotificationConnectionInfo(),
-      ).rejects.toThrow();
+      const err = await notification
+        .fetchNotificationConnectionInfo()
+        .catch((e) => e as BadRequestError);
+
+      expect(err).toBeInstanceOf(BadRequestError);
+      expect((err as BadRequestError).problemDetails.status).toBe(400);
+      expect((err as BadRequestError).problemDetails.title).toBeFalsy();
+      expect((err as BadRequestError).problemDetails.detail).toBeFalsy();
```

Comment:
Ditto on the assertions specificity.

Reply:
Addressed in https://github.com/inrupt/solid-client-notifications-js/pull/1031/commits/700073b7c4ab7d505b9a959a7cc00dafb5d85e37